app/index.js

```diff
@@ -50,6 +49,7 @@ const config = require('./config');
 
 config.init();
 
+const createMenu = require('./menu');
```

Comment:
Could you move it up there where it was before? There's no need for this change 🙌